src/components/pages/ReservationForm.jsx

```diff
@@ -0,0 +1,78 @@
+/* eslint-disable react/forbid-prop-types */
+import React, { useState } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import PropTypes from 'prop-types';
+import { createReservations } from '../../redux/reservation/reservationSlice';
+import './reservationForm.css';
+import SidebarComponents from '../sideBar/sideBarComponent';
+
+function ReservationForm({ car }) {
```

Comment:
- [ ] I was able to reserve in the past (I used the frontend to do so):

![image](https://github.com/Chandan-devs-tech/drive-easy-frontend/assets/57421823/32f96906-4984-4025-a794-4d65cea6b506)

Kindly, add some validation either in the frontend or in the server so your users can only reserve in the future. 👍